worker/runtime/backend.go

```diff
@@ -230,9 +332,11 @@ func (b *GardenBackend) createContainer(ctx context.Context, gdnSpec garden.Cont
 		return nil, fmt.Errorf("getting uid and gid maps: %w", err)
 	}
 
-	oci, err := bespec.OciSpec(b.initBinPath, gdnSpec, maxUid, maxGid)
+	oci, err := bespec.OciSpec(b.initBinPath, b.seccompProfile, b.ociHooks, gdnSpec, maxUid, maxGid)
 	if err != nil {
 		return nil, fmt.Errorf("garden spec to oci spec: %w", err)
+	} else {
+		fmt.Println("Using OCI spec", oci)
```

Comment:
What's the preferred setup for logging. Personally I'd prefer more verbose logging at debug level to cover both paths.

Reply:
At this level there is no a systematic logging we can use unforturnately. I am not too familiar with worker runtime code but it seems like only using error for debugging i.e. when there is unexpected behaviour, it will be wrap as error and return. 

FYI we will need to cut release v7.9 this week, if not is it ok to push this PR to 7.9.1?

Reply:
That's fine, no need to rush it, I want to do it proper.

Reply:
My main concern is around debugability, currently one cannot verify the correct passthrough of the oci runtime args without the logs. It's one of the more simple approaches handling.